src/index.css

```diff
@@ -161,17 +162,53 @@ nav a.active-link {
   display: none;
```

Comment:
I can't comment on it directly, but in the `.menu-button-container` style declaration block:
https://github.com/alisedarta/personal-project/blob/c63d0ebdbd0159b37a8c6ffcadc3594d6e90626e/src/index.css#L149-L152

You should remove the left & right padding, because it now adds on top of the container margin.

Reply:
Same thing further above: https://github.com/alisedarta/personal-project/blob/c63d0ebdbd0159b37a8c6ffcadc3594d6e90626e/src/index.css#L129-L135

```diff
- padding: 30px 16px 66px;
+ padding: 30px 0 66px;
```